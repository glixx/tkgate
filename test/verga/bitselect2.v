module top;
  reg [63:0] b;
  reg [7:0] c;
  integer i;

  initial
    begin
      $display("b1=%016h",b);
      b = 64'h0;
      $display("b2=%016h",b);
      b[40 +: 8] = 8'hff;
      $display("b3=%016h",b);
      b = 64'h0;
      b = 64'h0;
      $display("b4=%016h",b);
      b[44 +: 8] = 8'hff;
      $display("b5=%016h",b);
    end

endmodule
