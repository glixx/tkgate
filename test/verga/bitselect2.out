b1=xxxxxxxxxxxxxxxx
b2=0000000000000000
b3=0000ff0000000000
b4=0000000000000000
b5=000ff00000000000
